### PR TITLE
Test price

### PR DESCRIPTION
**Pull request by manycoding**

### CODE REVIEW

File: tests/rules/test_price.py [hunk 1]
```diff
@@ -124,3 +124,42 @@ def test_compare_names_for_same_urls(
         pd.DataFrame(source_data), pd.DataFrame(target_data), tagged_fields
     )
     assert result == create_result("Compare Names Per Url", expected_messages)
+
+
+@pytest.mark.parametrize(
```
Comment (manycoding) [inline tests/rules/test_price.py]: @WarleyLopes Can you read this test case?

Comment (WarleyLopes) [reply tests/rules/test_price.py]: @manycoding, I couldn't at the time but now I was able to understand it until the part it entered https://github.com/scrapinghub/arche/blob/master/src/arche/rules/price.py which I still get lost

Comment (manycoding) [reply tests/rules/test_price.py]: Ok, so you understand how this test works and test cases?
The `compare_prices_for_same_names` is too complex (and outdated), don't pay much attention to it.

Comment (manycoding) [reply tests/rules/test_price.py]: Ok, so you understand how this test works and test cases?
The `compare_prices_for_same_names` is too complex (and outdated), don't pay much attention to it.

Comment (WarleyLopes) [reply tests/rules/test_price.py]: All right. I'd say not to its entirety but I understand the coding, the part where you set data as an input for the test and the definition of the test itself if I'm not wrong of course

**Pull request merged**